Makefile

```diff
@@ -108,20 +108,25 @@ ifeq (,$(wildcard $(DEV_SSH_SERVER_KEY)))
 	ssh-keygen -t ed25519 -N '' -f $(DEV_SSH_SERVER_KEY) < /dev/null
 endif
 
-.PHONY: start-dev
-start-dev: builder teardown-dev $(DEV_SSH_SERVER_KEY)
-	docker run -d \
-		--name collector_remote_dev \
-		--cap-add sys_ptrace -p127.0.0.1:$(LOCAL_SSH_PORT):22 \
-		-v $(DEV_SSH_SERVER_KEY):/etc/sshkeys/ssh_host_ed25519_key:ro \
+.PHONY: start-builder
+start-builder: builder teardown-builder
+	docker run -id --entrypoint /bin/sh \
+		--name $(COLLECTOR_BUILDER_NAME) \
+		-v $(CURDIR):$(CURDIR) \
+		-w $(CURDIR) \
+		--cap-add sys_ptrace \
 		quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG)
 
-.PHONY: teardown-dev
-teardown-dev:
-	-docker rm -fv collector_remote_dev
+.PHONY: check-builder
+check-builder:
```

Comment:
Would it be handy to actually start the builder image if it's not there, to not require this manual step from a developer? On the CI it's being done for us, as far as I see.

Reply:
I can probably add it to the `check-builder.sh` script, I didn't see it as a big deal because I already spin up my own devcontainer for it anyways, but there's no harm in adding it I think.